luadoc build - Lua 5.3 unit tests failure

```bash
/usr/bin/lua5.3: ./helpers.lua:29: attempt to call a string value (field 'version')
stack traceback:
	./helpers.lua:29: in function 'need_lpeg_version'
	./c_tokenise.lua:22: in main chunk
	[C]: in upvalue 'fn'
	../CorsixTH/Lua/strict.lua:66: in function 'require'
	/home/runner/work/CorsixTH/CorsixTH/LDocGen/main.lua:47: in main chunk
	[C]: in ?
make[3]: *** [CMakeFiles/doc_corsixth_lua.dir/build.make:71: CMakeFiles/doc_corsixth_lua] Error 1
make[2]: *** [CMakeFiles/Makefile2:250: CMakeFiles/doc_corsixth_lua.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:231: CMakeFiles/doc.dir/rule] Error 2
make: *** [Makefile:169: doc] Error 2
```

Not sure what requires patching here.

--- a/LDocGen/c_tokenise.lua
+++ b/LDocGen/c_tokenise.lua
@@ -18,8 +18,7 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE. --]]
 
-local lpeg = require "lpeg"
-need_lpeg_version(0, 9)
+local lpeg = require("lpeg")
 
 local C, P, R, S, V = lpeg.C, lpeg.P, lpeg.R, lpeg.S, lpeg.V
 local Carg, Cc, Cp, Ct = lpeg.Carg, lpeg.Cc, lpeg.Cp, lpeg.Ct

--- a/LDocGen/helpers.lua
+++ b/LDocGen/helpers.lua
@@ -25,16 +25,6 @@ if not rawget(_G, "loadstring") then
   loadstring = load
 end
 
-function need_lpeg_version(major, minor)
-  local v = require"lpeg".version()
-  local m, n = v:match"^(%d+)%.(%d+)"
-  m = tonumber(m)
-  n = tonumber(n)
-  if m < major or (m == major and n < minor) then
-    error(("LPEG version >= %d.%d is required (you have %s)"):format(major, minor, v))
-  end
-end
-
 local auto_tokens
 local auto_tokens_c = {
   [","]  = {"operator", ","},

--- a/LDocGen/lua_tokenise.lua
+++ b/LDocGen/lua_tokenise.lua
@@ -18,8 +18,7 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE. --]]
 
-local L = require "lpeg"
-need_lpeg_version(0, 9)
+local L = require("lpeg")
 
 local C, P, R, S, V = L.C, L.P, L.R, L.S, L.V
 local Carg, Cc, Cp, Ct, Cg, Cb, Cmt = L.Carg, L.Cc, L.Cp, L.Ct, L.Cg, L.Cb, L.Cmt

